islandora.module

```diff
@@ -1962,8 +1962,7 @@ function islandora_islandora_metadata_display_info() {
  * Implements hook_islandora_datastream_access().
  */
 function islandora_islandora_datastream_access($op, AbstractDatastream $datastream, $user) {
-  module_load_include('inc', 'islandora', 'includes/utilities');
```

Comment:
Should usually be available when this is run, but... paranoia has me wanting to keep an include around for [the `islandora_invoke_hook_list()` call below](https://github.com/Islandora/islandora/pull/685/files#diff-ab8875f550771e37425e399a32097fb0R1971)... Could move it down beside include of `includes/derivatives`?

Reply:
Yes of course! How did it even work without that? I guess it was included somewhere up the chain, but thanks for the catch.

Reply:
Yeah, in the vast majority of cases, we'd be [somewhere under an `islandora_invoke_hook_list()` invocation already](https://github.com/Islandora/islandora/blob/9b37dcc9b853e109ce5ba3e1ae889703803ca3a5/islandora.module#L1734-L1738) (so it must be loaded)... The issue would be if somebody had/wanted to call this somewhere else.